src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -2761,17 +2769,36 @@ class AccessibilityHelpWidget extends Widget implements ITerminalWidget {
 	}
 
 	private _buildContent(): void {
-		const content = [introMessage];
+		const content = [];
 		content.push(this._descriptionForCommand(TerminalCommandId.EnterAccessibilityMode, enterAccessibilityModeNls, enterAccessibilityModeNoKb));
 		if (this._hasShellIntegration) {
 			content.push(shellIntegration);
 			content.push('- ' + this._descriptionForCommand(TerminalCommandId.RunRecentCommand, runRecentCommand, runRecentCommandNoKb));
 			content.push('- ' + this._descriptionForCommand(TerminalCommandId.GoToRecentDirectory, goToRecent, goToRecentNoKb));
 		}
 		content.push(this._descriptionForCommand(TerminalCommandId.OpenDetectedLink, openDetectedLink, openDetectedLinkNoKb));
-		content.push(readMore, dismiss);
-		const joinedContent = content.join('\n\n');
-		this._contentDomNode.domNode.appendChild(renderFormattedText(joinedContent));
-		this._contentDomNode.domNode.setAttribute('aria-label', joinedContent);
+		content.push(readMoreLink, dismiss);
+		for (const c of content) {
+			const element = c === readMoreLink ? renderElementAsMarkdown(this._markdownRenderer, this._openerService, readMoreLink, this._register(new DisposableStore())) : renderFormattedText(c);
+			if (c === readMoreLink) {
+				element.classList.add('terminal-help-link');
+				const link: HTMLElement | null = element.querySelector('p a');
+				if (link) {
+					link.tabIndex = 0;
+				}
```

Comment:
does not work without this

Reply:
also this make it such that the screen reader cannot detect the link (other than via tab)

Reply:
it still doesn't get recognized as a link by the screen reader with this markdown approach